📝 add docusaurus documentation (#266)
ajnart/homarr: docs/docs/customizations/custom-title.md

---
tags:
  - Customization
  - Design
  - Title
  - Custom
---

# Customize Page Title

> You can rename the default "Homarr" to whatever you'd like.

Reviewer: ```suggestion
You can change the page title to whatever you like.
```

> Open your settings, located on the top right, and switch to the tab "Customizations".

Reviewer: ```suggestion
Open settings by clicking the settings cog on the top right, and switch to the tab "Customizations".
```
